app/models/variable_lookup_key.rb

```diff
@@ -0,0 +1,18 @@
+class VariableLookupKey < LookupKey
```

Comment:
that would require changing the name to LookupKey::Variable or something, wouldn't it? I'd really prefer not to because it kind of creates a cyclic requirement (Foo::Bar < Foo)